components/Dashboard.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <div>
```

Comment:
If we keep the component and there will be more elements joining `header`, then yes, we'll want to keep the wrapper `div`.